docs/component-guidance/inputter.md

```diff
@@ -0,0 +1,251 @@
+### Anatomy of the component
+
+![Anatomy of the inputter](../images/inputter/inputter-anatomy.png)
+
+#### Label
+
+Every single `muon-inputter` must have a label to identify the form control and should describe its purpose – which helps users understand. See https://www.w3.org/WAI/tutorials/forms/labels/
```

Comment:
An input needs a label, but the inputter does not (see headings)

Reply:
```suggestion
An input needs a label to identify the form control and should describe its purpose – which helps users understand. See https://www.w3.org/WAI/tutorials/forms/labels/
```